### ISSUE TITLE
Events added to the array are not rendered

### ISSUE THREAD

**Issue by kjakobson** [richardtallent/vue-simple-calendar#6]
When I push event to the **events** array, it is not rendered. When I remove event it works fine.

**Comment by richardtallent** [comment]
Hmm... I'm limited to my phone for a few days, can you put together an example and I'll look into it when I'm back in civilization?

**Comment by richardtallent** [comment]
I'm back to a computer, created a test to push a new event, and did not see an issue, using code like this:

```Javascript
this.events.push({ id: 'e12', startDate: this.thisMonth(22), endDate: this.thisMonth(23), title: 'New Event' });
```

(This is using the sample app's `thisMonth()` utility function.)

Two potential issues I can think of before we try to create a more detailed test case:

1. Does your event have both a "startDate" and "endDate"? The calendar doesn't currently support a blank endDate (it should, so if that's the problem, we can convert this to a bug report for that).

2. When you created your dates, can you confirm you used a 0-based month number in your `new Date()` call? That always trips me up, and would result in your new event being created, but rendered on the month after the one intended.

**Comment by kjakobson** [comment]
I will check both 1. and 2. when I'm back at my work pc. It's actually possible I forgot to convert the date from ISO format which is sent by my API. I think you can close this issue for now.

**Comment by richardtallent** [comment]
I just released 1.5.0, which cleans up a few things, I went ahead and added support to make endDate optional (defaults to start), as well as id (defaults to random) and title (default "Untitled"). Will close for now.
